pokemongo_bot/__init__.py

```diff
@@ -265,7 +265,7 @@ def _register_events(self):
         )
         self.event_manager.register_event(
             'pokemon_evolved',
-            parameters=('pokemon', 'iv', 'cp')
+            parameters=('pokemon', 'iv', 'cp', 'xp')
```

Comment:
@Anakin5 you added a required parameter (no optional parameter logic available yet) to `pokemon_evolved` event here, but the EvolvePokemon task doesn't send it, so an error will happen. You need to make EvolvePokemon send it.